csa/_misc.py

```diff
@@ -181,29 +181,29 @@ def iterator (self, low0, high0, low1, high1, state):
                                        state)
         try:
             pre = []
-            (i, j) = maskIter.next ()
+            (i, j) = next(maskIter)
             while True:
                 # collect connections in one connection matrix column
                 post = j
                 while j == post:
                     pre.append (i)
-                    (i, j) = maskIter.next ()
+                    (i, j) = next(maskIter)
```

Comment:
Formatting

Reply:
I'm now stopping commenting about space before argument list---please fix it everuwhere.